SimPEG/Mesh/CylMesh.py

```diff
@@ -1,12 +1,11 @@
 import numpy as np
 import scipy.sparse as sp
 from scipy.constants import pi
-from SimPEG.Utils import mkvc, ndgrid, sdiag, kron3, speye, spzeros, ddx, av, avExtrap
+from SimPEG import Utils
```

Comment:
sorry, this makes this not an overly transparent pr, but is a helpful clean-up for code readability. I will highlight the important aspects with comments